internal/cluster/store/fsm.go

```diff
@@ -0,0 +1,134 @@
+package store
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/hashicorp/raft"
+	"github.com/pbinitiative/zenbpm/internal/cluster/command/proto"
+	pb "google.golang.org/protobuf/proto"
+)
+
+// FSM is Finite State Machine of the system state
+type FSM struct {
+	s *Store
```

Comment:
can we rename please? One-letter variable names should be very local scope only (see Clean Code book) 
```suggestion
	store *Store
```